scripts/lint-dependencies.js

```diff
@@ -47,7 +47,7 @@ const ignoredDependencies = new Set([
 ]);
 
 const regexps = [
-    /import\s+.*?\s+'([a-z0-9_\-@/.]+)';/gi, // `import * as something from 'something';`
+    /import[\s\w\d{},*]*?'([a-z0-9_\-@/.]+)';/gi, // `import * as something from 'something';`
```

Comment:
Even though the following will match:

```
import {{{{ something,
         another}}
       }}}}}}} from 'package';
```

I think it is not worth making it more accurate as the code will not compile.

Reply:
If the issue is caught by the compiler, I wouldn't waste time on a more accurate Regex.